pkg/backup/item_backupper.go

```diff
@@ -147,6 +147,22 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 		return nil
 	}
 
+	resource := obj.UnstructuredContent()
+	if conditions, err := collections.GetSlice(resource, "status.conditions"); err != nil {
```

Comment:
I think checking for `metadata.deletionTimestamp != nil` would be more universal - WDYT?

Reply:
yes it'll be easier.